tutorials/README.md

```diff
@@ -9,6 +9,7 @@
 | fileIO-simple     | Simple example to copy data from files in an input directory to rolling files in an output directory |
 | fileOutput        | Show how to use partitioning on both the file output operator and the input operator |
 | fileToJdbc        | Shows how to read files from HDFS, parse into POJOs and then insert into a table in MySQL.  |
+| formatter         | Shows how to use CsvFormatter to customise the delimiters and define the schema string. |
```

Comment:
Rename it to CSVFormatter as there are other formatters as well OR you can follow similar strategy like parsers, where we have all parser examples under single directory
https://github.com/DataTorrent/examples/tree/master/tutorials/parser